src/runtime/components/BlogDetailHeader.vue

```diff
@@ -49,7 +49,7 @@
             <span>{{ published_on }}</span> ·{{ " " }}
             <span> {{ post.readingTime }} min read</span>
 
-            <span v-if="post.published_on == null">
+            <span v-if="post.publishedAt == null">
```

Comment:
MAke sure you are overriding merged MR changes, if its required

Reply:
![image](https://github.com/canopas/canopas-blog-components/assets/140039768/db6fda0f-b82f-4d65-89f2-b1a4a4485e01)
its used to display the draft tag but if we use published_on then its not displayed